src/test/java/redis/clients/jedis/csc/JedisPooledClientSideCacheTest.java

```diff
@@ -32,26 +34,50 @@ public class JedisPooledClientSideCacheTest {
 
   protected Jedis control;
 
+  protected static final EndpointConfig sslEndpoint = HostAndPorts.getRedisEndpoint("standalone0-tls");
+
+  protected Jedis sslControl;
+
+  @BeforeClass
+  public static void prepare() {
+    setupTrustStore();
+  }
+
+  static void setupTrustStore() {
+    setJvmTrustStore("src/test/resources/truststore.jceks", "jceks");
+  }
+
+  private static void setJvmTrustStore(String trustStoreFilePath, String trustStoreType) {
+    assertTrue(String.format("Could not find trust store at '%s'.", trustStoreFilePath),
+        new File(trustStoreFilePath).exists());
+    System.setProperty("javax.net.ssl.trustStore", trustStoreFilePath);
+    System.setProperty("javax.net.ssl.trustStoreType", trustStoreType);
+  }
```

Comment:
Instead of having just one test method with SSL, do all tests with SSL in a new test class; i.e. SSSJedisPooledClientSideCacheTest.

Reply:
this is just POC of workaround(PING stuff) for SSL. Target is to replace it with solution that underlying socket utilized for availability check. But your comment is valid anyway.